service/firewall/packet_handler.go

```
@@ -580,6 +585,70 @@
 	issueVerdict(conn, pkt, 0, true)
 }
 
+func inspectDNSPacket(conn *network.Connection, pkt packet.Packet) {
+	// Ignore info-only packets in this handler.
+	if pkt.InfoOnly() {
+		return
+	}
+
+	dnsPacket := new(dns.Msg)
+	pkt.LoadPacketData()
```

golangci-lint (1.61.0)

Error return value of `pkt.LoadPacketData` is not checked

(errcheck)